src/sidebar/services/annotations-exporter.ts

```diff
@@ -23,7 +23,7 @@ export class AnnotationsExporter {
   }
 
   buildExportContent(
-    annotations: Annotation[],
+    annotations: APIAnnotationData[],
```

Comment:
Not specific to this PR, but the existence of `Annotation` vs `APIAnnotationData` and when to use each is likely to cause some confusion for future developers. Simply adding some notes on the type definition to clarify the conceptual difference (raw annotation from API vs augmented annotation with client-internal properties) would help here.

Reply:
Yep. In fact we typed as `Annotation` to be as strict as possible, while it actually should be the other way around. Accept the type that has the minimum amount of props you want, and filter-out the rest.

Reply:
Done https://github.com/hypothesis/client/pull/5782